perun-core/src/main/java/cz/metacentrum/perun/core/impl/GroupsManagerImpl.java

```diff
@@ -232,6 +232,18 @@ public Group updateGroup(PerunSession sess, Group group) throws InternalErrorExc
 			}
 			dbGroup.setDescription(group.getDescription());
 		}
+
```

Comment:
I would prefer implementing own method, instead of using updateGroup(), since some people use our API directly and they could create inconsistency by changing ``parentGroupId``, but not doing the rest of logic.

Reply:
Regarding future necessity of audit message about moving group, you should probably create ``updateParentGroupId()`` in BL layer too, where audit message will be logged in a future. Then use BL version from ``moveGroup()``